app/src/hooks/api/useProjectApi.ts

```diff
@@ -105,6 +105,18 @@ const useProjectApi = (axios: AxiosInstance) => {
     return data;
   };
 
+  /**
+   * Get public (published) project details based on its ID for viewing purposes.
+   *
+   * @param {number} projectId
+   * @return {*} {Promise<IGetProjectForViewResponse>}
+   */
+  const getPublicProjectForView = async (projectId: number): Promise<IGetProjectForViewResponse> => {
```

Comment:
Doesn't need to happen now, but it might be nice to nest these public functions under some sort of `public` object.

So that we can do something like `biohubApi.projects.public.getProjectForView()` or `biohubApi.public.projects.getProjectForView()`

That way we don't need to include the word `public` in every function name, and its a bit clearer which you are calling.

Reply:
cool!